samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/evaluation/HistogramListener.java

```diff
@@ -1737,7 +1770,10 @@ private void getCalculatedQuestionScores(List<ItemGradingData> scores, Histogram
 		LinkedHashMap<String, String> mainvariablesWithValues = new LinkedHashMap<>();
 		int total = 0;
 		int i = 1;
-		Long publishAnswerIdAnt = scores.get(0).getPublishedAnswerId();
+		Long publishAnswerIdAnt = 0L;
+		if (!scores.isEmpty()) {
```

Comment:
I've just noticed that this problem was resolved here on master.
https://github.com/sakaiproject/sakai/commit/61e24b0e7d7aa63a0c3181cecd922e0a2824676a
https://sakaiproject.atlassian.net/browse/SAK-48786

Similar solution this one than the patch on master.